### PR TITLE
Add aff-reattempt and webdriver

### PR DESCRIPTION
**Pull request by khanage**
Adds aff-reattempt and webdriver

### CODE REVIEW

File: packages.json [hunk 1]
```diff
@@ -32,6 +32,14 @@
     "repo": "https://github.com/nwolverson/purescript-aff-promise.git",
     "version": "v1.0.0"
   },
+  "aff-reattempt": {
+    "dependencies": [
+      "aff",
+      "refs"
+    ],
+    "repo": "git://github.com/slamdata/purescript-aff-reattempt.git",
```
Comment (LiamGoodacre) [inline packages.json]: Is there a reason why this one is `git://` and not `https://`?

Comment (khanage) [reply packages.json]: Whoops!

**Comment by khanage** [comment]
Is there anything else I need to do to get this PR merged?

**Comment by justinwoo** [comment]
LGTM

**Pull request merged**